frontend/src/views/reading_view.js

```diff
@@ -8,24 +8,56 @@ class Segment extends React.Component {
         return (
             <div className="scroll">
                 <p>Segment Number: {this.props.segment_num + 1}</p>
-                {this.props.segmentLines.map((line, k) => (
+                {this.props.segment_lines.map((line, k) => (
                     <p key={k}>{line}</p>)
                 )}
             </div>
         )
     }
 }
 Segment.propTypes = {
-    segmentLines: PropTypes.array,
+    segment_lines: PropTypes.array,
     segment_num: PropTypes.number,
 };
 
 
+class OverviewWindow extends React.Component {
+    render() {
+        return (
+            <div className={"row"}>
+                <div className={"col-8"}>
+                    <div className="scroll_overview">
+                        {this.props.all_segments.map((el, i) => (
+                            <p key={i}>{el.text}</p>)
+                        )}
+                    </div>
+                </div>
+                <div className={"col-4"}>
+                    <p><b>Document Questions</b></p>
+                    {this.props.document_questions.map((el, i) => (
+                        <p key={i}>{el.is_overview_question ? null : el.text}</p>)
```

Comment:
probably for both of these maps (overview questions and not) you'd want to do the conditional one level higher than just whether to show the text or not -- as is, you're littering the DOM with a bunch of empty `<p>` nodes